### DISCUSSION TITLE
Use a dummy index.qmd to point to the preferred document to render/use in a folder structure

### DISCUSSION THREAD

**Discussion by hectormz** [quarto-dev/quarto-cli#7299] [Feature Requests]
### Description

I keep a structure for my site that looks something like:

```
.
└── blog/
    ├── post1/
    │   ├── notebooks/
    │   │   └── notebook.ipynb
    │   └── data
    └── post2/
        ├── notebooks/
        │   └── notebook.ipynb
        └── data
```

but this creates a url like: `www.mysite.com/blog/post1/notebooks`

instead I would like `www.mysite.com/blog/post1`

Is there a way to use a `.qmd` to specify which document to render, but keep the structure as if the `.qmd` was the document itself?

```
.
└── blog/
    ├── post1/
    │   ├── index.qmd
    │   ├── notebooks/
    │   │   └── notebook.ipynb
    │   └── data
    └── post2/
        ├── index.qmd
        ├── notebooks/
        │   └── notebook.ipynb
        └── data
```
In this case, `index.qmd` is a mostly empty file that just has a header that points to the nested `.ipynb`.

I'm aware of [`includes`](https://quarto.org/docs/authoring/includes.html) but it doesn't seem to be what I'm looking for.

I think I could make an alias/redirect, but I really just want the post to live at `www.mysite.com/blog/post1`

**Comment by mcanouil** [comment]
For short, no you can't get what you want with your structure.
Watch #6518 for future development.

Meanwhile (and as already discussed in some of the issues inside #6518, especially #2065), you can use aliases, see <https://quarto.org/docs/websites/website-navigation.html#redirects>

**Reply by hectormz** [reply]
Thanks @mcanouil !
